tests/Tests.AzureAppConfiguration/FeatureManagementTests.cs

```diff
@@ -1129,5 +1144,66 @@ Response<ConfigurationSetting> GetTestKey(string key, string label, Cancellation
         {
             return Response.FromValue(TestHelpers.CloneSetting(FirstKeyValue), new Mock<Response>().Object);
         }
+
+        [Fact]
+        public void WithRequirementType()
```

Comment:
It got missed during the first review, but can you move the test up before private methods `GetIfChanged` and `GetTestKey`.

Reply:
Oh right, I think I changed all these in my old PR but was basing off the reverted changes again. I'll fix these, thanks